### PR TITLE
Fixes DevDiv #736427

### PR DESCRIPTION
**Pull request by rynowak**
When a user builds a 2.1 or 2.2 project with a 3.0 they can get an error
due to invalid command line options from rzc. What happens is that the
3.0 tasks (which are a singleton) are talking to the 2.X build tool
(which is NOT a singleton).

Recently some changes were make to the 3.0 tasks which caused this bug
because it did not properly accomodate downlevel versions of rzc. We
have an integration test for this scenario for but the regression was
not detected, so obviously something is wrong with the test setup. I'll
investigate that separately.

### CODE REVIEW

File: src/Microsoft.NET.Sdk.Razor/RazorGenerate.cs [hunk 1]
```diff
@@ -90,6 +90,14 @@ protected override string GenerateResponseFileCommands()
 
             builder.AppendLine(Command);
 
+            // We might be talking to a downlevel version of the command line tool, which doesn't
+            // understand certain parameters. Assume 2.1 if we can't parse the version because 2.1
+            // 2.2 are the releases that have command line tool delivered by a package.
```
Comment (NTaylorMullen) [inline src/Microsoft.NET.Sdk.Razor/RazorGenerate.cs]: `and 2.2`

**Review by NTaylorMullen — APPROVED** [review]
:shipit:

**Comment by rynowak** [comment]
Closing in favor of https://github.com/aspnet/AspNetCore/pull/4290

**Review by ajaybhargavb — COMMENTED** [review]
:shipit:

**Pull request closed without merging**